README.md

```diff
@@ -3,25 +3,26 @@
 ![Linear Img](http://mikaa123.github.io/linear-website/images/screenshot.png)
 
 Linear is a ruler application for Mac, made with web development in mind.
-It sits in your menu bar and doesn't get in your way. Here is what it does:
+It sits in your menu bar and doesn't get in your way. Here are a few highlights:
```

Comment:
I changed `Here is what it does` to `Here are a few highlights` because the first entry on the list doesn't describe something the app *does*, rather, it describes something that the app *isn't* (a browser extension).